docs/dev-guide/contributing/building.rst

```diff
@@ -8,54 +8,54 @@ Concepts
 ^^^^^^^^
 
 There are some concepts you should internalize before you begin making builds. Koji has a concept
-called tags. A tag is essentially a grouping of package builds, and it maps to a Yum repository.
-Pulp uses one Koji tag per Pulp release stream, per distribution, per architecture. Pulp uses three
-release streams per release, corresponding to development, testing, and production. In Koji, these
-three concepts map to "testing", "beta", and "" (empty string) in our tag names respectively.
-
-This table maps the git branches to their appropriate Koji tags:
-
-+---------------+-----------------------------------+
-| Git Branch    | Koji Tag per <distribution>       |
-+===============+===================================+
-| <X.Y>-dev     | pulp-<X.Y>-testing-<distribution> |
-+---------------+-----------------------------------+
-| <X.Y>-testing | pulp-<X.Y>-beta-<distribution>    |
-+---------------+-----------------------------------+
-| <X.Y>-release | pulp-<X.Y>-<distribution>         |
-+---------------+-----------------------------------+
-
-.. warning::
-
-   Note the potential confusion that the X.Y-testing branch maps to the pulp-X.Y-beta-rhel7 tag, not
-   the pulp-X.Y-testing-rhel7 tag. It could be wise for us to resolve this by renaming our Koji
-   tags to correspond to our dev, testing, and release branch names, but at this time this is how
-   they map.
-
-For example, the 2.4 release stream in Pulp platform has three git branches, 2.4-dev, 2.4-testing,
-and 2.4-release. For RHEL 7, these map to pulp-2.4-testing-rhel7, pulp-2.4-beta-rhel7, and
-pulp-2.4-rhel7, respectively. You can see the full list of Pulp's Koji tags
+called tags. A tag is essentially a grouping of package builds.
+Pulp uses one Koji tag per Pulp X.Y release stream, per distribution, per architecture.
+For example, the 2.6 releases of pulp will build into the pulp-2.6-<distribution> tags in koji.
+You can see the full list of Pulp's Koji tags
```

Comment:
Not so much formally decided so much as that with the new assembly method they just were no longer providing value.